src/node-web-compat-web.ts

```diff
@@ -56,60 +60,59 @@ export const nodeWebCompat: NodeWebCompat = {
       "Synchronously transforming a JWK into a key object is not supported in the browser"
     );
   },
-  transformJwkToKeyObjectAsync: (jwk: RsaSignatureJwk) =>
-    window.crypto.subtle.importKey(
+  transformJwkToKeyObjectAsync: (jwk, jwtHeaderAlg) => {
```

Comment:
Saw it is superfluous since the overall object where this is in, is already typed as `NodeWebCompat` which does the job. So I removed the superfluous type declarations.